tests/testprogs/uprobe_fork_loop.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+int uprobeFunction1(int *n, char c __attribute__((unused)))
+{
+  return *n;
+}
+
+void spin()
+{
+  while (1)
+  {
+    int n = 13;
+    char c = 'x';
+    uprobeFunction1(&n, c);
+    usleep(500);
+  }
+}
+
+int main(int argc __attribute__((unused)), char **argv __attribute__((unused)))
+{
+  pid_t p = fork();
```

Comment:
Should we wait for the child? Otherwise zombie process right

Reply:
We want them to be running at the same time.

Reply:
Ah yeah. Thought `spin()` had finite iterations. If we rely on runtime test runner to kill the process group this should be fine